src/main/java/com/google/api/codegen/viewmodel/StaticLangStubSettingsView.java

```diff
@@ -51,6 +51,19 @@
     return unaryCallSettings;
   }
 
+  /** ApiCalls that send exactly one request. This includes unary and server streaming apis. */
+  public List<ApiCallSettingsView> unaryRequestCallSettings() {
```

Comment:
What exactly does the `request` part of this mean, and how does it distinguish this case from `unaryCallSettings`? The name doesn't feel right because it has `unary` in it, and it returns settings for server streaming methods.

Reply:
a. unary rpcs are 1:1 request/response
b. server streaming are 1:n
c. client streaming are n:1
d. bidi are n:m

I was trying group cases a & b "RPCs that send exactly one request before receiving N responses". Would `singleRequestRpcCallSettings` be better?

Naming is hard :(